server.js

```diff
@@ -4840,6 +4840,52 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 
+// Test if a webpage is online integration
+camp.route(/^\/website(-(([^-]|--)*?)-(([^-]|--)*)(-(([^-]|--)+)-(([^-]|--)+))?)?\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var onlineMessage = escapeFormat(match[2] != null ? match[2] : "online");
+  var offlineMessage = escapeFormat(match[4] != null ? match[4] : "offline");
+  var onlineColor = escapeFormat(match[7] != null ? match[7] : "brightgreen");
+  var offlineColor = escapeFormat(match[9] != null ? match[9] : "red");
+  var userURI = match[11];
+  var format = match[12];
+  var withProtocolURI = userURI.match(/^[^:\/]+:/) ? userURI : ("http://" + userURI);
```

Comment:
@espadrine The problem is that the protocol then becomes mandatory, so you can't just write the short form `/website/mysite.com`. Should I make the protocol mandatory?